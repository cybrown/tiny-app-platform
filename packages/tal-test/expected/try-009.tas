missing_variable catch { "value on error" }
