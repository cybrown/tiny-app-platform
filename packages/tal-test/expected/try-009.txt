(1:1) Unknown symbol: missing_variable
any
